backend/src/question/question.service.ts

```diff
@@ -1,18 +1,36 @@
 import { Injectable } from "@nestjs/common";
+import { Cron } from "@nestjs/schedule";
 import * as _ from "lodash";
 
-import { FlattenedQuestionContent, FlattenedQuestionSummary } from "shared/api";
+import {
+  FlattenedQuestionContent,
+  FlattenedQuestionSummary,
+  NAMESPACES,
+} from "shared/api";
+import {
+  QUESTION_CONTENT,
+  QUESTION_QOTD_SUMMARY,
+  QUESTION_QOTD_CONTENT,
+  QUESTION_SUMMARIES,
+  QUESTION_TOPICS,
+} from "./question.cache.keys";
 import {
   QuestionContentFromDb,
   QuestionSummaryFromDb,
   QUESTION_CONTENT_SELECT,
   QUESTION_SUMMARY_SELECT,
 } from "./question.type";
 import { PrismaService } from "../prisma/prisma.service";
+import { RedisCacheService } from "../cache/redisCache.service";
 
 @Injectable()
 export class QuestionService {
-  constructor(private prisma: PrismaService) {}
+  constructor(private prisma: PrismaService, private cache: RedisCacheService) {
+    // Invalidate cache on launch, for demo/test purposes
+    (async () => {
+      await this.invalidateQuestionCache();
+    })();
```

Comment:
actually can just restart the redis docker to clear the cache right?

Reply:
@aidanaden in case we wanna test in deployment also, currently it invalidates on the 15th minute of the hour every hour